commons/src/main/java/org/calrissian/accumulorecipes/commons/hadoop/BaseQfdInputFormat.java

```diff
@@ -72,6 +74,15 @@ protected static void configureScanner(Job job, Node query, GlobalIndexVisitor g
         setting.addOption(BooleanLogicIterator.QUERY_OPTION, originalJexl);
         setting.addOption(BooleanLogicIterator.FIELD_INDEX_QUERY, jexl);
 
+        final String[] selectFields = job.getConfiguration().getStrings("selectFields");
+
+        setting.addOption("auths", getScanAuthorizations(job).serialize());
```

Comment:
This can actually be removed for now- I missed this when I was refactoring the original pull request.